kilkari/src/main/java/org/motechproject/nms/kilkari/domain/WeeklyCallsNotAnsweredMsisdnRecord.java

```diff
@@ -0,0 +1,51 @@
+package org.motechproject.nms.kilkari.domain;
+
+import org.joda.time.DateTime;
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.Unique;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
+/**
+ * Record of Weekly Calls Not Answered Deactivated Numbers
+ */
+@Entity(tableName = "nms_weekly_calls_not_answered_msisdn")
```

Comment:
It is named that way since we are saving the numbers that are requested by MOHFW and as they wanted the reason to be weekly calls not answered. I will change it to "blocked_msisdn" and ideally the last 'deactivation_reason' column will have the only reason "weekly_calls_not_answered" until MOHFW comes up with new reason.